### PR TITLE
UT-Earthworks-12d-1

### PR DESCRIPTION
**Pull request by LeeGregory12d**
## Unit test

one earthworks fill object

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [x] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: UT-Earthworks-12d-1/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has an earthworks fill object.
+
+The Ifc file was generated by 12d Model.
```
Comment (pjanck) [inline UT-Earthworks-12d-1/readme.md]:
```suggestion
The IFC file was generated by 12d Model. 
```

**Review by pjanck — APPROVED** [review]
Simple, yet very nice example for Earthworks.

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

File: UT-Earthworks-12d-1/readme.md (hunk 1 above)
Comment (LeeGregory12d) [reply UT-Earthworks-12d-1/readme.md]: Fixed when added the RC3 example in 6717bf2

**Review by AlexBrad1eyCT — APPROVED** [review]
I see no problem with the unit test after RC3 upgrade Good to Merge

**Pull request merged**